.github/dependabot.yml

```diff
@@ -9,3 +9,8 @@ updates:
       prefix-development: chore
       include: scope
     open-pull-requests-limit: 5
+    ignore:
```

Comment:
I don't really know what I'm doing in this file, I'm just pattern matching from the [Dependabot docs page](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#specifying-dependencies-and-versions-to-ignore)

Reply:
My take from the docs page is that this will make it so DBot treats it as though the `@nx` and `@nrwl` packages don't exist in the project, and will not do anything whatsoever with them... neither report that updates are needed, nor create update PRs.

I think it's a stronger action than what we actually want, but by my read of the docs it doesn't really give us an intermediate option.

Reply:
The [`groups` feature](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#groups) might be workable?

If, say, we insist that all `@nx` and `@nrwl` updates should be grouped into single PRs?

This working correctly would rely on the assumption that dependabot would 'notice' **all** of the relevant updates all at once. I could see this still glitching if the grouped update PR only included a subset of the `@nx`/`@nrwl` packages that are specified.

.... that said, we now know about having to keep all of them in sync (which we should document somewhere ... in README?), and this `groups` feature _should_ at least cluster all of these version bumps into a single PR...?

Reply:
I was just about to write and say thanks for asking about how we're going to get updates for `nx` because then I took a second look and also noticed the groups feature. Update incoming